_data/simple-icons.json

```diff
@@ -5314,6 +5314,12 @@
             "source": "https://fidoalliance.org/overview/legal/logo-usage/",
             "guidelines": "https://fidoalliance.org/overview/legal/fido-trademark-and-service-mark-usage-agreement-for-websites/"
         },
+        {
+            "title": "FIDO Passkey",
+            "hex": "E7AC22",
+            "source": "https://fidoalliance.org/overview/legal",
+            "guidelines": "https://fidoalliance.org/get-the-passkey-icon"
```

Comment:
Hmm I've found https://fidoalliance.org/overview/legal/logo-usage/, https://fidoalliance.org/get-the-passkey-icon/, https://fidoalliance.org/terms-of-use/, https://fidoalliance.org/fido-logo-trademark-usage-agreement-for-press-and-educational-purposes/, and https://fidoalliance.org/overview/legal/ which all seem like plausible places to link to. Probably should determine which of these pages seems the most authoritative or links to the most of the other pages?

Reply:
The guidelines referenced in the PDF link to [the FIDO Alliance legal overview](https://fidoalliance.org/overview/legal). This page also contain links to other relevant legal documents. 

https://fidoalliance.org/get-the-passkey-icon might be more efficient for specific passkey information. But we can definitely use the legal overview page